src/contracts/core/StrategyManager.sol

```diff
@@ -612,6 +563,7 @@ contract StrategyManager is
         onlyStrategiesWhitelistedForDeposit(strategy)
         returns (uint256 shares)
     {
+        require(address(strategy) != address(eigenPodManager.beaconChainETHStrategy()), "StrategyManager._depositIntoStrategy: cannot deposit into the canonical beaconChainETHStrategy via the StrategyManager");
```

Comment:
this is unnecessary, not sure why it was added

Reply:
yup

Reply:
OK so this will revert anyway when the address with no code doesn't return any value, so going to remove it as it's unnecessary

Reply:
OK so this function will revert anyway when the address with no code doesn't return any value, so I think I'm just going to delete it.

Reply:
yeah fair enough, I thought it might be good to throw an informative error but im just overthinking it